include/circt/Dialect/SV/SVStatements.td

```diff
@@ -438,6 +438,28 @@ def VerbatimOp : SVOp<"verbatim"> {
   ];
 }
 
+def FileOp : SVOp<"file", [SingleBlock, NoTerminator, NoRegionArguments,
```

Comment:
I'm a bit confused about which direction you're going here.  Are you planning to keep sv.file, or go with the attribute approach?

Reply:
Attribute approach, forgot to switch to draft mode though so you don't see the interim pushes. Sorry for the spam!